README.md

```diff
@@ -101,6 +101,11 @@ Be aware that Cloud Shell has [several limitations][cloud-shell-limitations],
 in particular an inactivity timeout that will close running shells after 20
 minutes. Please consider it only for blueprints that are quickly deployed.
 
+## VM Image Support
+
+The HPC Toolkit supports the HPC CentOS 7 VM image and the Ubuntu 20.04 LTS
```

Comment:
nit: I would turn this into a list as I expect it to grow over time and it will make it more readable.

Reply:
Done. Also added a table of contents in the vm-images doc that will help when the list gets bigger.